_includes/header.html

```diff
@@ -2,7 +2,14 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ "/" | relative_url}}">{{ site.title | escape }}</a>
+    <div class="logo-title">
+      <img class="site-logo" src="/assets/header/mmh_logo_blue.png"/>
+
+
+        <a class="site-title" href="{{ "/" | relative_url}}">{{ site.title | escape }}</a>
+
```

Comment:
Would you be willing to clean up the blank lines and indentation of these new lines?